Build instruction for Ubuntu missing cc

https://bedrockdb.com/#ubuntu-linux

When running those on a Ubuntu 18 via WSL (in WindowS), you get this error:

```
sujan@Surface-Book-2:~/bedrock/Bedrock$ make
git submodule init
Submodule 'mbedtls' (https://github.com/ARMmbed/mbedtls.git) registered for path 'mbedtls'
git submodule update
Cloning into '/home/sujan/bedrock/Bedrock/mbedtls'...
Submodule path 'mbedtls': checked out '04a049bda1ceca48060b57bc4bcf5203ce591421'
cd mbedtls && git checkout -q 04a049bda1ceca48060b57bc4bcf5203ce591421
cd mbedtls && make no_test && touch library/libmbedcrypto.a && touch library/libmbedtls.a && touch library/libmbedx509.a
make[1]: Entering directory '/home/sujan/bedrock/Bedrock/mbedtls'
make[2]: Entering directory '/home/sujan/bedrock/Bedrock/mbedtls/library'
  CC    aes.c
make[2]: cc: Command not found
Makefile:195: recipe for target 'aes.o' failed
make[2]: *** [aes.o] Error 127
make[2]: Leaving directory '/home/sujan/bedrock/Bedrock/mbedtls/library'
Makefile:18: recipe for target 'lib' failed
make[1]: *** [lib] Error 2
make[1]: Leaving directory '/home/sujan/bedrock/Bedrock/mbedtls'
make: *** No rule to make target 'mbedtls/library/libmbedcrypto.a', needed by 'libstuff/libstuff.d'.  Stop.
```

Can be fixed with `sudo apt-get install build-essential` per https://help.duo.com/s/article/3603?language=en_US. After that, `make` succeeds.

--- a/docs/index.md
+++ b/docs/index.md
@@ -40,7 +40,7 @@ You can build from scratch as follows:
     # Install some dependencies
     sudo add-apt-repository ppa:ubuntu-toolchain-r/test
     sudo apt-get update
-    sudo apt-get install gcc-9 g++-9 libpcre++-dev zlib1g-dev
+    sudo apt-get install build-essential gcc-9 g++-9 libpcre++-dev zlib1g-dev
 
     # Build it
     cd Bedrock

